core/extraction/towns_fund.py

```diff
@@ -437,6 +437,8 @@ def extract_funding_questions(df_input: pd.DataFrame, programme_id: str) -> pd.D
     fund_questions_df["Response"].replace("< Select >", "", inplace=True)
     fund_questions_df["Programme ID"] = programme_id
     fund_questions_df = fund_questions_df.reset_index(drop=True)
+    if programme_id.split("-")[0] == "HS":
+        fund_questions_df.drop(fund_questions_df.index, inplace=True)
```

Comment:
If a user is uploading for a FHSF we don't even need to run the function that extracts and transforms funding question data. We can skip this entire line of code:

```
    towns_fund_extracted["Funding Questions"] = extract_funding_questions(
        df_ingest["4a - Funding Profiles"],
        programme_id,
    )
```

Reply:
Yeah, I agree with this. Only issue is, correct me if I'm wrong, the validator expects an empty sheet with the correct columns to be there so we would need to return a dummy sheet and this was the simplest way I could implement that behaviour

Reply:
you could potentially add an optional param to the funding question extraction function, defaulted to False, something like `return_empty_sheet=False`, then add a line right at the start of the function that just returns an empty df with the correct columns? 

This would save on unnecessary computation.

Reply:
Have you treid the Pndas .isempty method?
`assert df.empty`

Reply:
Or (apparently) even faster:
`assert len(df.index) > 0:`